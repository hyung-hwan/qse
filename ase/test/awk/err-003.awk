global abc;

BEGIN {
	abc[20] = "abc";
	abc = 10;
}
