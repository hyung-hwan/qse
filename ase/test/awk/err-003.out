global abc;

BEGIN {
	abc[20] = "abc";
	abc = 10;
}

RUN ERROR: CODE [90] LINE [5] map 'abc' not assignable with a scalar
